app/views/layouts/design_system.html.erb

```diff
@@ -0,0 +1,38 @@
+<% environment = GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment %>
+
+<%= render "govuk_publishing_components/components/layout_for_admin", product_title: "Publisher", browser_title: yield(:page_title), environment: environment  do %>
+
+  <% render "layouts/google_tag_manager" %>
+
+  <%= render "govuk_publishing_components/components/skip_link" %>
+
+  <%= render "govuk_publishing_components/components/layout_header", {
+    product_name: "Publisher",
+    environment: environment,
+    navigation_items: navigation_items(is_editor: current_user.govuk_editor?, path: request.path, user_name: current_user.name)
+  } %>  
```

Comment:
There is some trailing blank spaces on this line.

Reply:
This blank space is still present (I only noticed as it causes an issue when I rebase my branch on top of this one, as my editor automatically deletes trailing blank space).